sparse_strips/vello_common/src/strip.rs

```diff
@@ -0,0 +1,203 @@
+// Copyright 2025 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Rendering strips.
+
+use crate::footprint::Footprint;
+use crate::peniko::Fill;
```

Comment:
I'd prefer bringing in Peniko as a crate dependency to keep the import paths slightly simpler (and allow grouping external and internal imports). As it's versioned at the workspace-level already, there shouldn't be any issues. The re-exports in `vello_api` are a convenience for consumers and can be useful if they have multiple versions of, say, Kurbo.